src/app/components/profile/profile.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+    <div id="profile" *ngIf="newUser.userId">
+        <h2>Profile</h2>
+        <table class="table table-hover">
+            <tr>
+                <th scope="col">User Name</th>
+                <td><input type="text" name="userName" [(ngModel)]="newUser.userName" [readonly]="editable!=='userName'" [ngClass]="editable==='userName' ? 'editField' : ''"></td>
+                <td><img class="edit" (click)="edit('userName')" src="../../../assets/pencil.png"/></td>
+            </tr>
+            <small *ngIf="newUser.userName && !validateUserName()">Invalid Username</small>
+            <tr>
+                <th scope="col">First Name</th>
+                <td><input type="text" name="firstName" [(ngModel)]="newUser.firstName" [readonly]="editable!=='firstName'" [ngClass]="editable==='firstName' ? 'editField' : ''"></td>
+                <td><img class="edit" (click)="edit('firstName')" src="../../../assets/pencil.png"/></td>
+            </tr>
+            <small *ngIf="newUser.firstName && !validateName(newUser.firstName)">Invalid Name</small>
+            <tr>
+                <th scope="col">Last Name</th>
+                <td><input type="text" name="lastName" [(ngModel)]="newUser.lastName" [readonly]="editable!=='lastName'" [ngClass]="editable==='lastName' ? 'editField' : ''"></td>
+                <td><img class="edit" (click)="edit('lastName')" src="../../../assets/pencil.png"/></td>
+            </tr>
+            <small *ngIf="newUser.lastName && !validateName(newUser.lastName)">Invalid Name</small>
+            <tr>
+                <th scope="col">Email</th>
+                <td><input type="text" name="email" [(ngModel)]="newUser.email" [readonly]="editable!=='email'" [ngClass]="editable==='email' ? 'editField' : ''"></td>
+                <td><img class="edit" (click)="edit('email')" src="../../../assets/pencil.png"/></td>
```

Comment:
Assets shouldn't use relative paths. "assets/pencil.png" should render correctly and follows best practices